src/urg_node_driver.cpp

```diff
@@ -686,22 +691,22 @@ void UrgNode::run()
   if (publish_multiecho_)
   {
     echoes_freq_.reset(new diagnostic_updater::HeaderlessTopicDiagnostic("Laser Echoes",
-          *diagnostic_updater_,
+          diagnostic_updater_,
           FrequencyStatusParam(&freq_min_, &freq_min_, diagnostics_tolerance_, diagnostics_window_time_)));
   }
   else
   {
     laser_freq_.reset(new diagnostic_updater::HeaderlessTopicDiagnostic("Laser Scan",
-          *diagnostic_updater_,
+          diagnostic_updater_,
           FrequencyStatusParam(&freq_min_, &freq_min_, diagnostics_tolerance_, diagnostics_window_time_)));
   }
 
-  // Now that we are setup, kick off diagnostics.
-  close_diagnostics_ = false;
-  diagnostics_thread_ = std::thread(std::bind(&UrgNode::updateDiagnostics, this));
+  //// Now that we are setup, kick off diagnostics.
+  //close_diagnostics_ = false;
+  //diagnostics_thread_ = std::thread(std::bind(&UrgNode::updateDiagnostics, this));
 
   // Start scanning now that everything is configured.
   close_scan_ = false;
-  scan_thread_ = std::thread(std::bind(&UrgNode::scanThread, this));
+  //scan_thread_ = std::thread(std::bind(&UrgNode::scanThread, this));
```

Comment:
Accidentally commented?

Reply:
Most certainly. I could see myself doing that while trying to make this compile without having a physical device at hand.